sympy/concrete/summations.py

```diff
@@ -269,9 +269,77 @@ def _eval_difference_delta(self, n, step):
 
         return Sum(f, (k, upper + 1, new_upper)).doit()
 
-    def _eval_simplify(self, ratio, measure):
-        from sympy.simplify.simplify import sum_simplify
-        return sum_simplify(self)
+    def _eval_simplify(self, ratio=1.7, measure=None):
+        from sympy.simplify.simplify import sum_add
+        from sympy.core.exprtools import factor_terms
+        from sympy import Mul
+        s = self
+
+        terms = Add.make_args(s)
+        s_t = [] # Sum Terms
+        o_t = [] # Other Terms
+
+        for term in terms:
+            if isinstance(term, Mul):
+                other = 1
+                sum_terms = []
+                for j in range(len(term.args)):
+                    if isinstance(term.args[j], Sum):
+                        sum_terms.append(term.args[j]._eval_simplify())
+                    else:
+                        other = other * term.args[j]
+                if len(sum_terms):
+                    # remove constant from the Sum
+                    s_t.append(other * Mul(*sum_terms))
+                else:
+                    o_t.append(term)
+            elif isinstance(term, Sum):
+                constant = 1
+                inner = 1
+                sum_vars = set([sum_var.args[0] for sum_var in term.args[1:]])
+                subterms = Mul.make_args(factor_terms(term.args[0]))
+                sum_terms = []
+                for j in range(len(subterms)):
+                    theterm = subterms[j]
```

Comment:
You only use `j` to index `subterms` so you could just use `for theterm in subterms`.